File: C/pointers/b_array/p5.c
#include<stdio.h>
int main(){
	int a = 10, b = 20, c = 30, d = 40;

	int* arr[5]={&a,&b,&c,&d,NULL};


	for(int i=0;i<5;i++){
		printf("%d\n",**(arr+i));

	}





}

File: java/CRUD/p5.java
//converting collection to array

import java.util.*;

class Test{
	public static void main(String args[]){
		//List interface have toArray() method, so all list based collection can be converted to array using toArray() method
		
		Integer a[] = {1,5,625,3,9};

		ArrayList<Integer> l1 = new ArrayList<Integer>(Arrays.asList(a));

		Vector<Integer> l2 = new Vector<Integer>(l1);



		System.out.println(l2);

		Integer arr[] = l2.toArray(new Integer[0]);
	
		System.out.println(Arrays.toString(arr));


	}
}
